### ISSUE TITLE
Remove in-file package switches

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#1004]
I've just pushed an important fix to define-command:
a15ca6efa3e816490bc5ffe80d29b4d5ee04eff3.
The consequence is that now we can define globally-visible commands from non-nyxt packages and we don't need to switch to the `:nyxt` package anymore.

This is much cleaner I believe.

I suggest we remove all the in-file package switches.  There are a few, starting with web-move.

Thoughts?

**Comment by jmercouris** [comment]
Hard to say. Using in-package kind of makes the intent very clear. Defining nyxt::some-command maybe is not as obvious. I'm OK with either way. If you think this is cleaner, why not :-)

**Comment by Ambrevar** [comment]
It makes the intent clear if you read the whole file.

But it implies that context changes depending on code position within
the file, which can be confusing.  (By the way, it confuses SLIME / SLY.)

For instance

```
(in-package :nyxt)
(define-command foo ...)
```

is clear, while

```
(in-package :nyxt)

; 100 lines below

(define-command bar ...)
```

Is BAR a global command or a mode-local command?  The only way to know
it is to scan the file for `(in-package :nyxt)`.


What do you find ambiguous with nyxt::some-command?

**Comment by jmercouris** [comment]
It just wouldn't make me think, ah, they are defining something in the Nyxt package! It's not ambiguous, just non-obvious to me at first glance.

**Comment by Ambrevar** [comment]
Closed with bf6f3173259f0c79f473150d1937e32acf1dce30.
